core/utils/poly-look/src/visualisations/charts/bubble-charts/bubbleCluster/bubbleCluster.js

```diff
@@ -9,9 +9,11 @@ const bubblePadding = 3;
 const bigBubblesFont = "20px";
 const mediumBubblesFont = "16px";
 const defaultBubbleColor = "blue";
+const defaultStrokeColor = "#f7fafc";
 const defaultTextColor = "white";
 const defaultOpacity = 1;
-const defaultValueShowing = true;
+const defaultText = (d) =>
```

Comment:
I have a naming nit here, but I don't have good suggestions. Thing is, naming should indicate this is a closure, not a simple constant. `defaultTextOrNothing`? `computeDefaultText`?

Reply:
Those are actually changes pulled in from my changes - not sure why :D As for naming: We're following D3 conventions here, it can _either_ be a value, _or_ a closure. That goes for all of the defaults above.